test/inductor/test_torchinductor_opinfo.py

```diff
@@ -489,24 +534,35 @@ def fn(*args, **kwargs):
                 if device_type == "cuda":
                     # opinfo test case have already place the input on the correct device
                     # so we don't need do additional copy by setting copy_to_cuda=False
+                    adjusted_kwargs = {
+                        "check_lowp": False,
+                        "nopython": True,
+                        "copy_to_cuda": False,
+                        "reference_in_float": False,
+                        "check_gradient": requires_grad,
+                    }
+                    adjusted_kwargs.update(overridden_kwargs)
+
                     self.check_model_cuda(
                         fn,
                         args,
                         kwargs,
-                        check_lowp=False,
-                        nopython=True,
-                        copy_to_cuda=False,
-                        reference_in_float=False,
-                        **additional_kwargs,
+                        **adjusted_kwargs,
                     )
                 elif device_type == "cpu":
+                    adjusted_kwargs = {
+                        "check_lowp": False,
+                        "nopython": True,
+                        # skip checking gradient on CPU for now
```

Comment:
Why?

Reply:
Inductor with CPU is not the focus now... We can enable this when CPU becomes a priority.